src/main/webapp/map.html

```diff
@@ -8,15 +8,19 @@
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCEr8IFDb0oorKv9HmcL-XDPu62EAa8kRY&libraries=places&callback=initAutocomplete"
         async defer></script>
   </head>
-  <body onload="createMap();">
+  <!-- <body onload="createMap();"> -->
     <h1>Google Map API</h1>
-    <div id="map"></div>
-    <p>This page shows a Google Map using a fetch method to get data!</p>
+    <!-- <div id="map"></div> -->
```

Comment:
Try not to commit commented out code unless there's a reason for keeping it around (and then leave a comment explaining why it's there)